docs/en/releases/1.16.md

```diff
@@ -167,21 +167,21 @@ Please continue reading for [upgrade instructions](#upgrade-guide).
 This release contains a major rework for the rover support in PX4:
 
 - Complete restructure of the [rover related documentation](../frames_rover/index.md).
-- New firmware build specifically for [rovers](../frames_rover/index.md#flashing-the-rover-build).
-- New module dedicated to [Ackermann rovers](../frames_rover/ackermann.md):
-  - The module currently supports [manual mode](../flight_modes_rover/ackermann.md#manual-mode), [acro mode](../flight_modes_rover/ackermann.md#acro-mode), [position mode](../flight_modes_rover/ackermann.md#position-mode) and [auto modes](../flight_modes_rover/ackermann.md#auto-modes).
-- New module dedicated to [differential rovers](../frames_rover/differential.md):
-  - The module currently supports [manual mode](../flight_modes_rover/differential.md#manual-mode), [acro mode](../flight_modes_rover/differential.md#acro-mode), [stabilized mode](../flight_modes_rover/differential.md#stabilized-mode), [position mode](../flight_modes_rover/differential.md#position-mode) and [auto modes](../flight_modes_rover/differential.md#auto-modes).
-- New module dedicated to [mecanum rovers](../frames_rover/mecanum.md):
-  - The module currently supports [manual mode](../flight_modes_rover/mecanum.md#manual-mode), [acro mode](../flight_modes_rover/mecanum.md#acro-mode), [stabilized mode](../flight_modes_rover/mecanum.md#stabilized-mode), [position mode](../flight_modes_rover/mecanum.md#position-mode) and [auto modes](../flight_modes_rover/mecanum.md#auto-modes).
+- New firmware build specifically for [rovers](../config_rover/index.md#flashing-the-rover-build).
```

Comment:
FYI only, I'm hoping that we are able to manage docs builds properly for v1.16 and that these updates won't leak into that docs build. I'm not all that hopeful.